packit_service/worker/allowlist.py

```diff
@@ -440,14 +443,22 @@ def check_and_report(
             event, "user_login", None
         ) or getattr(event, "actor", None)
 
-        if user_login and user_login in service_config.admins:
+        if user_login and user_login in self.service_config.admins:
             logger.info(f"{user_login} is admin, you shall pass.")
             return True
 
         for related_events, callback in CALLBACKS.items():
             if isinstance(event, related_events):
-                return callback(event, project, service_config, job_configs)
+                return callback(event, project, job_configs)
 
         msg = f"Failed to validate account: Unrecognized event type {type(event)!r}."
         logger.error(msg)
         raise PackitException(msg)
+
+    def get_approval_issue(self, namespace) -> Optional[str]:
+        for issue in self.service_config.get_project(
+            url=NOTIFICATION_REPO
+        ).get_issue_list(author=self.service_config.get_github_account_name()):
+            if issue.title.strip().endswith(f" {namespace} needs to be approved."):
```

Comment:
I was curious if we have this info in the DB or you'd look for it.

Reply:
We agreed on doing it this way...;)